### DISCUSSION TITLE
Catching Firebird errors

### DISCUSSION THREAD

**Discussion by marcin-bury** [asfernandes/node-firebird-drivers#104] [Q&A]
Hi

First, please excucse my lack of JS/TS knowledge.
Is there a way to catch Firebird errors? Like incorrect login/password?
I'd like to return proper response to the web client when establishing connection was not possible due to inproper login credentials.

TIA
Marcin

**Comment by asfernandes** [comment]
All errors are currently thrown as `Error` object.

This is pending: https://github.com/asfernandes/node-firebird-drivers/issues/1
